data/brands/shop/clothes.json

```diff
@@ -7276,23 +7315,33 @@
       }
     },
     {
-      "displayName": "Takko",
-      "id": "takko-475eda",
+      "displayName": "Takko Fashion",
```

Comment:
I search taginfo and found there are more than 1k elements named with `Takko`, is this brand changed their name? If so, keep a matchNames is needed. Or many existing elements won't be matched.

Reply:
I made this change because the logo, storefronts and websites all talk about Takko Fashion with "fashion" part of the brand

https://company.takko.com/en-gb/newsroom/press-releases.html

So I think we should include it in the name. I though the match on qid would fix the name. 
but if there are any further changes needed the please ask